test/bdd/features/search.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Searching for assets on the DKG
+  Background: Setup local blockchain, bootstraps and nodes
+    Given the blockchain is set up
+    And 1 bootstrap is running
+
+  Scenario: Search assertion on the network with keywords
+    Given I setup 4 nodes
+    When I call search request on node 1 with result type assertions for the keywords:
+      | keyword 1 | keyword 2 |
+    And I wait for last search request to finalize
```

Comment:
Can we update the steps to be more uniform and to be the same as it is for resolve. To look something like this:
And I wait for last search request to finalize
And Last search finished with status: COMPLETED
And Last search returned valid result

You can merge all checks into one step: Last search returned valid result